enhancements/node-tuning/pao-in-nto.md

```diff
@@ -0,0 +1,317 @@
+
+### Non-Goals
+
+Logic changes or tighter integration with other MCO controllers are not expected to be part of this initial move. Just the delivery mechanism wrt OCP.
+
+## Proposal
+
+To be really clear here, the Telco team will handle all the necessary changes to both codebases, however we will need support in the form of PR reviews and design consultations.
+
+### Code
+
+PAO code would be moved into the NTO repository as a separate controller. The Telco team would keep ownership as it has the necessary domain knowledge needed to implement the low latency tuning.
+
+The code currently uses Operator SDK libraries in some of the tests and this might need to be changed.
+Migrating the use of these libaries to [client-go](https://github.com/openshift/library-go)
+would be one of the options.
+
+PAO also implements a [conversion webhook](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion)
+to migrate its custom resources during upgrade.
+The hook is currently installed via OLM mechanisms and this might need to be reimplemented or changed to use whatever CVO or ART support.
+PAO must-gather will have the native support with additional arguments and will not require an additional image as currently exists in PAO original repository.
+
+[cnf-tests](https://github.com/openshift-kni/cnf-features-deploy/) are NOT part of this move either, the Telco team will keep maintaining the validation suite in OpenShift KNI.
+
+
+### CI
+
+PAO contains both unit and functional tests implemented using Ginkgo.
+
+Integration with NTO is currently TBD, please advise if anything special is needed.
+
+One of the test suites (4_latency) implements an opinionated latency testing suite to get comparable results from customers and to prevent configuration / command line invocation mistakes. Keeping it in PAO/NTO or moving it into cnf-tests is TBD.
+
+### Build pipeline
+
+PAO would inherit the NTO pipeline and all the images will be released via ART.
+
+### Releases
+
+PAO should be shipped as part of core OCP after NTO will contain the migrated code of PAO.
+PAO already obeys the OCP github PR rules and uses the same PR bot so nothing changes in the lifecycle.
+
+### API stability
+
+The `PerformanceProfile` CR uses an `performance.openshift.io/v2` API group which marks it as stable interface that needs to be maintained for about a year after official deprecation. We do not expect any change to the API as part of this proposal.
+
+### Bugs
+
+PAO bugs are currently tracked in The OpenShift Bugzilla component [Performance Addon Operator](https://bugzilla.redhat.com/buglist.cgi?component=Performance%20Addon%20Operator&list_id=12053428&product=OpenShift%20Container%20Platform)
+
+This should be kept as-is, the Telco team will own all the bugs. We might need to move the component under NTO or rename it.
```

Comment:
I do think we're going to want to consider the bugzilla component question carefully. Moving to a sub-component of NTO seems like it makes sense, since that's the component users will see installed, but it would be good to have the teams decide what to do together.

Reply:
@MarSik  @jmencak  Do we want to move this to the open questions section or have a decision on that ?